extensions/amp-google-read-aloud-player/amp-google-read-aloud-player.md

```diff
@@ -0,0 +1,153 @@
+
+## Attributes
+
+<table>
+  <tr>
+    <td width="20%"><strong>data-api-key (required)</strong></td>
+    <td>API key identifies the publisher, and must be provided in order to use the player.
+<br><br>
+Each API Key is correlated to a closed list of domains that represent the publisher domains. Only
+URLs that belong to one of these domains are able to be played by the web player.
+<br><br>
+During the piloting phase, the API Key will be provided to the publisher by Google. Other API Keys
+that were generated using the Google Cloud Console won’t work.
+</td>
+  </tr>
+    <tr>
+    <td width="20%"><strong>data-tracking-ids (required)</strong></td>
+    <td>The list of the Google Analytics tracking IDs to send the player metrics to (comma
+separated).
+<br><br>
+You are required to have a Google Analytics tracking ID in order to use this player during the
+pilot period. The tracking ID will be provided to you by Google.
+<br><br>
+If you want to send the player metrics also to your Google Analytics account, add your tracking ID 
+(or IDs) to this list.
+</td>
+  </tr>
+  <tr>
+    <td width="20%"><strong>data-voice (required)</strong></td>
+    <td>The voice to use for generating the Text-to-Speech audio. Publishers should use only
+a single voice and only from the supported voices list (TBD).</td>
+  </tr>
+  <tr>
+    <td width="20%"><strong>data-url</strong></td>
+    <td>The canonical URL of the web page to read aloud. The URL must be publicly accessible and
+free of any PII. The URL must be on a domain that was configured by Google and correlated with the
+specific API key.
+<br><br>
+When the parameter isn’t provided, the player uses the Canonical URL link tag of the current page.
+This is useful for reading aloud the content of the current page and only a single Read Aloud
+player is embedded in the page (no infinite scroll).
+<br><br>
+This value overrides it, and publishers are required to set it only in these cases:
+<ol>
+  <li>Other page - When the player is embedded in a different page than the one that its content
+should be read aloud. In this case, set this parameter to the canonical URL of the other page.</li>
+  <li>Multiple players - When there are multiple players on the same page, see Multiple players
+(infinite scroll). In this case, set this parameter to the canonical URL of the article that the
+player should read aloud.</li>
+  <li>Canonical URL tag is missing - When the Canonical URL link tag of the current page is missing
+(and cannot be added). In this case, set this parameter to the canonical URL of the page.</td>
+</li>
+</ol>
+  </tr>
+  <tr>
+    <td width="20%"><strong>data-speakable</strong></td>
+    <td>Controls the contents to read aloud.
+<br><br>
+Possible values:
+<ul>
+  <li>false - Google automatically analyzes the web page and decides which text parts are suitable
+for reading aloud.</li>
+  <li>true - Google uses only text from elements marked with the speakable markup on the web page.
+</li></td>
```

Comment:
`true` and `false` are confusing since they're not string values. You could reword this to refer to attribute presence instead.

I would also add a documentation link that explains "speakable markup".

```suggestion
When present, uses text only from elements annotated with [speakable markup](https://...). Otherwise, Google analyzes the document automatically to choose the suitable text parts.
```